core/string/unpack/1_spec.rb

```diff
@@ -0,0 +1,12 @@
+require File.expand_path('../../../../spec_helper', __FILE__)
```

Comment:
I think it would be better to move this file under `core/string/unpack1_spec.rb`.
Under the unpack directory, it's specs by unpack directive but `1` is not a unpack directive.